pkg/koordlet/resourceexecutor/reader.go

```diff
@@ -38,6 +38,7 @@ type CgroupReader interface {
 	ReadMemoryNumaStat(parentDir string) ([]sysutil.NumaMemoryPages, error)
 	ReadCPUTasks(parentDir string) ([]int32, error)
 	ReadPSI(parentDir string) (*PSIByResource, error)
+	ReadMemoryIdlePageStatsByKidled(parentDir string) (*sysutil.ColdPageInfoByKidled, error)
```

Comment:
The naming suffix `-IdlePageStatsByKided` and the returned `sysutil.ColdPageInfoByKidled` involve too much detail about the kidled. As for the CgroupReader, we expect its methods are simply used by the caller regardless of the specific environments (e.g. cgroups v1 or v2, kidled or some other cold page technique). So there should be like `ReadColdPageUsage`, but not `ReadXXXKidledStatsByKidled` and `ReadXXXKstaledStatsByKstaled`.